### ISSUE TITLE
[Req-idea] make dashboard theme responsive 

### ISSUE THREAD

**Issue by AlternativeIndustries** [ParsePlatform/parse-dashboard#385]
Is possibile make the Dashboard theme resposive for mobile device? I am not able, but maybe someone know how do that. :-)

**Comment by drew-gross** [comment]
This would be pretty difficult, but we would be more than happy to accept help!

**Comment by pivanov** [comment]
I've play a lot with the dashboard and like Drew said it will be extremely difficult to do this. But if we can if we can identify the most important things that a user needs to do from mobile we can use react native to build an app maybe? Because I think the current dashboard works fine on tablet, right?

**Comment by MattiaConfalonieri** [comment]
@pivanov you right on tablet works fine... but on phone...

in my opinion the most important things are: filter class and edit objects.

Actually the big problem with the dashboard on phone is the impossibility to scroll horizontally.

**Comment by pivanov** [comment]
Yeah I know ... but as I said there are a lot of problems. It's not impossible but is very difficult. I made a lot of changes on DB Browser ... almost change everything ... and we have V and H scroll and a lot of other things ... the only one problem is that I can't send PR to the dashboard because we rewrite a lot of things (because of we have a lot or specific things with our infrastructure) but!!! when our implementation is stable it will be cool if we can merge our dashboard version or make some other things. So if you have some ideas it will be cool if I can help.

**Comment by MattiaConfalonieri** [comment]
Have the ability to see Past pushes from mobile is not a bad idea...
